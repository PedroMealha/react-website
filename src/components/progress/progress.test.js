import { shallow } from "enzyme";
import React from "react";
import Progress from "./progress.component";

describe("Progress component", () => {
  it("expect to render Progress component", () => {
    const mockSkill = {
      title: "ReactJs",
      percent: 35,
      learning: true,
    };

    expect(shallow(<Progress skill={mockSkill} />)).toMatchSnapshot();
  });
});
