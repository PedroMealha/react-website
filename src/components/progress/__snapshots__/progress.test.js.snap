// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress component expect to render Progress component 1`] = `
<div
  className="tool "
>
  <p>
    ReactJs
     
    <span>
      *
    </span>
  </p>
  <div
    className="progress"
  >
    <div
      style={
        Object {
          "width": "35%",
        }
      }
    >
      <div />
    </div>
  </div>
</div>
`;
